package/yast2-schema.spec

```diff
@@ -29,6 +29,8 @@ License:        GPL-2.0-or-later
 Url:            https://github.com/yast/yast-schema
 
 # Dependencies needed to build the package
+BuildRequires:  jing
```

Comment:
Um, this will pull in whole Java stack as a dependency. :worried:
I guess the OBS guys will not be happy about this...

I think it would be better to run the check regularly at Travis or Jenkins.

Or would it be enough just to use `xmllint`? That's much smaller...

Reply:
Yes it does pull in the JRE but we already must use Trang which also needs it, so it's not a new dependency.